Bernard/NamesController.swift

```diff
@@ -45,7 +45,7 @@ class NamesController : NamesControllerProtocol, NamesModelObserving {
         viewController?.nameText = namesModel.previousName()
     }
     
-    func viewDidLoad() {
+    func configureViewController() {
```

Comment:
`NamesController` is not a subclass of `UIViewController`, so it's misleading that it has a `viewDidLoad` method (it makes it seem as though there should be a call to `super.viewDidLoad` here). I've renamed this method to reflect what it's actually doing.